specification/paths/Returns-v1-shops-shop_id-reasons-return_reason_id.json

```
@@ -7,6 +7,113 @@
       "$ref": "#/components/parameters/path-return_reason_id"
     }
   ],
+  "patch": {
+    "tags": [
+      "ReturnReasons"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "returns.manage"
+        ]
+      }
+    ],
+    "summary": "Update a shop's return reason by ID.",
+    "description": "This endpoint updates a shop's return reason, allowing a shop to override some of the default attributes of a return reason and to attach (optional) questions to be answered when a consumer creates a return",
+    "requestBody": {
+      "description": "The shop return reason object with attributes to be updated.",
+      "required": true,
+      "content": {
+        "application/vnd.api+json": {
+          "schema": {
+            "type": "object",
+            "required": [
+              "data"
+            ],
+            "additionalProperties": false,
+            "properties": {
+              "data": {
+                "type": "object",
+                "properties": {
+                  "attachments": {
+                    "description": "Determines whether a consumer is allowed, required or forbidden to upload attachments related to the selected return reason",
+                    "example": "optional",
+                    "type": [
+                      "boolean",
+                      "string",
+                      "null"
+                    ],
+                    "enum": [
+                      false,
+                      "optional",
+                      "required"
```

Maybe a bit late to the party, but I suggest we get rid of the boolean possibility and replace this with:
```suggestion
                      "disabled",
                      "optional",
                      "required"
```